src/examples/box_with_concubes.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/python
+
+import json
+
+from ConCube import ConCube
+
+from cycax.cycad.assembly import Assembly
+from cycax.cycad.cuboid import Cuboid
+from cycax.cycad.engine import Engine
+from cycax.cycad.external_part import ExternalPart
+from cycax.cycad.sheet_metal import SheetMetal
+
+LEFT = "LEFT"
+RIGHT = "RIGHT"
+TOP = "TOP"
+BOTTOM = "BOTTOM"
+FRONT = "FRONT"
+BACK = "BACK"
+
+
+class main:
+    def main():
+        box = Assembly("box")
+
+        bottom = SheetMetal(x_size=100, y_size=100, z_size=2, part_no="bottom")
```

Comment:
FYI: Duplicate part_no.

The variables names for the SheetMetal instances could get one very confused. Consider nice descriptive names. Like box_top_side or box_side_top.  If someone else had to read this example they would be lost in 0.002s.

Reply:
Now that the example code works I see that it should be duplicated.